package.json

```diff
@@ -371,4 +371,3 @@
     ]
   }
 }
-
```

Comment:
Let me know if this change is problematic.
It has been added by my IDE when saving this file.
Some of the other packages `json` files have the ending empty line, but some do not. So, I wasn't sure how to approach it. 🤷